.github/workflows/stale.yml

```diff
@@ -0,0 +1,40 @@
+name: Stale Issue Closer
+permissions:
+  issues: write
+  
+on:
+  schedule:
+    #        ┌───────────── minute (0 - 59)
+    #        │ ┌───────────── hour (0 - 23)
+    #        │ │ ┌───────────── day of the month (1 - 31)
+    #        │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #        │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #        │ │ │ │ │
+    #        │ │ │ │ │
+    #        │ │ │ │ │
+    #        * * * * *
+    - cron: "0 0 * * *"
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v3
+        with:
+          repository: "microsoft/vscode-github-triage-actions"
+          path: ./actions
+          ref: stable
+      - name: Install Actions
+        run: npm install --production --prefix ./actions
+      - name: Run Stale Issue Closer
+        uses: ./actions/needs-more-info-closer
```

Comment:
Doesn't it need this action?

Reply:
I assume that action would be in the actions folder

Reply:
oh yeah, you probably need to copy the source here:
https://github.com/microsoft/vscode-github-triage-actions/tree/stable/needs-more-info-closer

Reply:
So you're saying I need to create an action folder in our repo and also copy the source to our repo? I am new to git actions so maybe I misunderstood something.🥲My understanding is that the `Checkout Actions` step checks out the microsoft/vscode-github-triage-actions repository at the stable branch to the ./actions directory within the current repository. Then the `Install Actions` step installs some dependencies in the ./actions directory. And the last step runs the local action file located at ./actions/needs-more-info-closer

Reply:
Oh sorry, I didn't notice that code. That sounds like it could work too.